spec.html

```diff
@@ -264,7 +264,7 @@ <h1>`Promise.any` Reject Element Functions</h1>
       1. Set _errors_[_index_] to _x_.
       1. Set _remainingElementsCount_.[[Value]] to _remainingElementsCount_.[[Value]] - 1.
       1. If _remainingElementsCount_.[[Value]] is 0, then
-        1. Perform ? ThrowAggregateError(_errors_).
+        1. Return ? Call(_promiseCapability_.[[Resolve]], undefined, &laquo; ThrowAggregateError(_errors_) &raquo;).
```

Comment:
I think the abstract operation returns an abrupt completion, so you’d need to construct the error without throwing it?

Reply:
You are right. In that case, I think, we don't need that abstract operation at all. I removed it in the next commit.